# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1343782007 14400
#      Tue Jul 31 20:46:47 2012 -0400
# Node ID b62b0b85369c2f1d09be7be657f909364c2796fe
# Parent  2cb76b2b8b299281a57c7e73767034d2dd982779
move more files to corefcn directory

* bitfcns.cc, mappers.cc, sparse.cc, strfns.cc, syscalls.cc: Move to
corefcn diretory.
* src/Makefile.am (DIST_SRC): Remove bitfcns.cc, mappers.cc,
sparse.cc, strfns.cc, and syscalls.cc from the list.
* src/corefcn/module.mk (COREFCN_SRC): Add bitfcns.cc, mappers.cc,
sparse.cc, strfns.cc, and syscalls.cc to the list

diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -233,17 +233,16 @@ nodist_octinclude_HEADERS = \
 JIT_SRC = \
   jit-util.cc \
   jit-typeinfo.cc \
   jit-ir.cc \
   pt-jit.cc
 
 DIST_SRC = \
   Cell.cc \
-  bitfcns.cc \
   c-file-ptr-stream.cc \
   comment-list.cc \
   cutils.c \
   data.cc \
   debug.cc \
   defaults.cc \
   defun.cc \
   dirfns.cc \
@@ -263,17 +262,16 @@ DIST_SRC = \
   ls-ascii-helper.cc \
   ls-hdf5.cc \
   ls-mat-ascii.cc \
   ls-mat4.cc \
   ls-mat5.cc \
   ls-oct-ascii.cc \
   ls-oct-binary.cc \
   ls-utils.cc \
-  mappers.cc \
   matherr.c \
   mex.cc \
   oct-fstrm.cc \
   oct-hist.cc \
   oct-iostrm.cc \
   oct-lvalue.cc \
   oct-map.cc \
   oct-obj.cc \
@@ -284,22 +282,19 @@ DIST_SRC = \
   oct-strstrm.cc \
   octave.cc \
   pager.cc \
   pr-output.cc \
   procstream.cc \
   profiler.cc \
   sighandlers.cc \
   siglist.c \
-  sparse.cc \
   sparse-xdiv.cc \
   sparse-xpow.cc \
-  strfns.cc \
   symtab.cc \
-  syscalls.cc \
   sysdep.cc \
   token.cc \
   toplev.cc \
   txt-eng-ft.cc \
   unwind-prot.cc \
   utils.cc \
   variables.cc \
   xdiv.cc \
diff --git a/src/bitfcns.cc b/src/corefcn/bitfcns.cc
rename from src/bitfcns.cc
rename to src/corefcn/bitfcns.cc
diff --git a/src/mappers.cc b/src/corefcn/mappers.cc
rename from src/mappers.cc
rename to src/corefcn/mappers.cc
diff --git a/src/corefcn/module.mk b/src/corefcn/module.mk
--- a/src/corefcn/module.mk
+++ b/src/corefcn/module.mk
@@ -5,16 +5,17 @@ COREFCN_SRC = \
   corefcn/__contourc__.cc \
   corefcn/__dispatch__.cc \
   corefcn/__lin_interpn__.cc \
   corefcn/__pchip_deriv__.cc \
   corefcn/__qp__.cc \
   corefcn/balance.cc \
   corefcn/besselj.cc \
   corefcn/betainc.cc \
+  corefcn/bitfcns.cc \
   corefcn/bsxfun.cc \
   corefcn/cellfun.cc \
   corefcn/colloc.cc \
   corefcn/conv2.cc \
   corefcn/daspk.cc \
   corefcn/dasrt.cc \
   corefcn/dassl.cc \
   corefcn/det.cc \
@@ -35,36 +36,40 @@ COREFCN_SRC = \
   corefcn/hess.cc \
   corefcn/hex2num.cc \
   corefcn/inv.cc \
   corefcn/kron.cc \
   corefcn/lookup.cc \
   corefcn/lsode.cc \
   corefcn/lu.cc \
   corefcn/luinc.cc \
+  corefcn/mappers.cc \
   corefcn/matrix_type.cc \
   corefcn/max.cc \
   corefcn/md5sum.cc \
   corefcn/mgorth.cc \
   corefcn/nproc.cc \
   corefcn/pinv.cc \
   corefcn/quad.cc \
   corefcn/quadcc.cc \
   corefcn/qz.cc \
   corefcn/rand.cc \
   corefcn/rcond.cc \
   corefcn/regexp.cc \
   corefcn/schur.cc \
+  corefcn/sparse.cc \
   corefcn/spparms.cc \
   corefcn/sqrtm.cc \
   corefcn/str2double.cc \
   corefcn/strfind.cc \
+  corefcn/strfns.cc \
   corefcn/sub2ind.cc \
   corefcn/svd.cc \
   corefcn/syl.cc \
+  corefcn/syscalls.cc \
   corefcn/time.cc \
   corefcn/tril.cc \
   corefcn/typecast.cc
 
 noinst_LTLIBRARIES += corefcn/libcorefcn.la
 
 corefcn_libcorefcn_la_SOURCES = $(COREFCN_SRC)
 
diff --git a/src/sparse.cc b/src/corefcn/sparse.cc
rename from src/sparse.cc
rename to src/corefcn/sparse.cc
diff --git a/src/strfns.cc b/src/corefcn/strfns.cc
rename from src/strfns.cc
rename to src/corefcn/strfns.cc
diff --git a/src/syscalls.cc b/src/corefcn/syscalls.cc
rename from src/syscalls.cc
rename to src/corefcn/syscalls.cc
